substrate/primitives/runtime/src/testing.rs

```diff
@@ -283,139 +239,22 @@ where
 	}
 }
 
-/// The signature payload of a `TestXt`.
-type TxSignaturePayload<Extra> = (u64, Extra);
-
-impl<Extra: TypeInfo> SignaturePayload for TxSignaturePayload<Extra> {
-	type SignatureAddress = u64;
-	type Signature = ();
-	type SignatureExtra = Extra;
-}
-
-/// Test transaction, tuple of (sender, call, signed_extra)
-/// with index only used if sender is some.
-///
-/// If sender is some then the transaction is signed otherwise it is unsigned.
-#[derive(PartialEq, Eq, Clone, Encode, Decode, TypeInfo)]
-pub struct TestXt<Call, Extra> {
```

Comment:
Could we keep this and make it a type alias for backwards compatibility?

Reply:
It needs to change because the old type isn't compatible with general transactions, so we would have to break downstream users anyway. I added the type alias back and if it is only used on construct runtime, it will just work. However, if downstream users want to access fields like `signature` in the old `TestXt`, they will have to update their code. Worth noting that this is just a test type.

Reply:
The only down side of a type alias to `CheckedExtrinsic` is that it doesn't implement `Checkable` trait.

But it can be fine.